vscode/webviews/Chat.tsx

```diff
@@ -231,6 +247,12 @@ export const Chat: React.FunctionComponent<React.PropsWithChildren<ChatboxProps>
                 eventName: 'CodyVSCodeExtension:chatActions:reset:executed',
                 properties: { source: 'chat', eventType },
             })
+            postMessage?.({
+                command: 'recordEvent',
+                feature: 'chatActions:reset',
+                action: 'executed',
```

Comment:
1. Feature must prefix with `cody.`
2. Verb should be the `action`, not the `feature`

```suggestion
                feature: 'cody.chatActions',
                action: 'reset',
```

Reply:
Good call! I'm going to rename these to:
feature: `cody.chatActions`
action: `reset:executed`
just because we have some actions that are either submitted or executed :)

Reply:
Or `reset.executed`, per the naming scheme?

Reply:
ah yes!!